### ISSUE TITLE
minOccurs default should be treated as `1`

### ISSUE THREAD

**Issue by zolvaczi** [tefra/xsdata#831]
```
	<xs:complexType name="DataPDU">
		<xs:sequence>
			<xs:element name="Revision" type="xs:string" fixed="2.0.14" />
			<xs:element name="Header" type="Header" />
  ...
		</xs:sequence>
	</xs:complexType>
```

generates

```
@dataclass
class DataPdu:
    class Meta:
        name = "DataPDU"
        namespace = "urn:swift:saa:xsd:saa.2.0"

    revision: str = field(
        init=False,
        default="2.0.14",
        metadata={
            "name": "Revision",
            "type": "Element",
            "required": True,
        }
    )
    header: Optional[Header] = field(
        default=None,
        metadata={
            "name": "Header",
            "type": "Element",
            "required": True,
        }
    )
...
```
`header` should not be `Optional`. The default value for `minOccurs` in the XSD is `1`.

**Comment by zolvaczi** [comment]
Hello,

Any idea why the `Message` element of `Header` is still generated as `Optional`?

```
	<xs:complexType name="Header">
		<xs:choice>
			<xs:element name="Message" type="Message" />
			<xs:element name="TransmissionReport" type="TransmissionReport" />
			<xs:element name="DeliveryNotification" type="DeliveryNotification" />
			<xs:element name="DeliveryReport" type="DeliveryReport" />
			<xs:element name="HistoryReport" type="HistoryReport" />
			<xs:element name="MessageStatus" type="MessageStatus" />
			<xs:element name="SessionStatus" type="SessionStatus" />
		</xs:choice>
	</xs:complexType>
	<xs:complexType name="DataPDU">
		<xs:sequence>
			<xs:element name="Revision" type="xs:string" fixed="2.0.14" />
			<xs:element name="Header" type="Header" />
			<xs:element name="Body" type="SwAny" minOccurs="0" />
			<xs:element name="LAU" type="SwAny" minOccurs="0" />
		</xs:sequence>
	</xs:complexType>
	<!-- Root element -->
	<xs:element name="DataPDU" type="DataPDU"></xs:element>
</xs:schema>
```

```
@dataclass(kw_only=True)
class DataPdu:
    class Meta:
        name = "DataPDU"
        namespace = "urn:swift:saa:xsd:saa.2.0"

    revision: str = field(
        init=False,
        default="2.0.14",
        metadata={
            "name": "Revision",
            "type": "Element",
            "required": True,
        }
    )
    header: Header = field(
        metadata={
            "name": "Header",
            "type": "Element",
            "required": True,
        }
    )
    ...

@dataclass(kw_only=True)
class Header:
    message: Optional[Message] = field(
        default=None,
        metadata={
            "name": "Message",
            "type": "Element",
            "namespace": "urn:swift:saa:xsd:saa.2.0",
        }
    )
    ...
```

**Comment by tefra** [comment]
Hi @zolvaczi  elements inside `<xs:choice minOccurs="1" maxOccurs="1'>` means that only one of them can appear.

If you want to group choice elements in a single field you can enable the compound fields
https://xsdata.readthedocs.io/en/latest/examples/compound-fields.html
